fvtest/compilertest/tests/OptionSetTest.hpp

```diff
@@ -29,7 +29,7 @@
 
 namespace TestCompiler {
 
-typedef std::map<const std::string, int> MethodSets;
+typedef std::map<std::string, int> MethodSets;
```

Comment:
I think a previous commit in this PR added the `const`. Could the history be changed so the `const` isn't added in the first place?

Reply:
I don't think this is the case. The `const` exists in the master branch [1].

[1] https://github.com/eclipse/omr/blob/051896be071812db8a3b71485b014be26c80e7f4/fvtest/compilertest/tests/OptionSetTest.hpp#L32

Reply:
Hm. You're right. I must be misremembering something.